en/installconfig-hwe-kernels.md

```diff
@@ -58,25 +100,25 @@ perform these three configurations from the CLI.
 
 ### Default minimum kernel
 
-To set the default minimum HWE kernel for all machines visit the 'Settings' page
-and select a HWE kernel in the 'Default Minimum Kernel Version' field. Don't
+To set the default minimum kernel for all machines visit the 'Settings' page
+and select a kernel in the 'Default Minimum Kernel Version' field. Don't
 forget to click 'Save'.
 
 ![image](../media/default_min_hwe_kernel.png)
 
 ### Machine minimum kernel
 
-To set the minimum HWE kernel on a machine basis visit the machine's page and click
-the `Edit` button in the 'Machine summary' pane. Then select a HWE kernel in the
+To set the minimum kernel on a machine basis visit the machine's page and click
+the `Edit` button in the 'Machine summary' pane. Then select a kernel in the
 'Minimum Kernel' field and 'Save changes'.
 
 ![image](../media/min_hwe_kernel.png)
 
 ### Machine kernel during deployment
 
-To set a specific HWE kernel during deployment visit the machine's page and choose
-`Deploy` under 'Take action'. Then select a HWE kernel in the 'Default
-kernel' field. Hit 'Go' to initiate the deployment.
+To set a specific kernel during deployment visit the machine's page and choose
```

Comment:
As above, maybe making it clearer that the machine's page is accessed from the Nodes view.